src/cache/cache.jl

```diff
@@ -28,6 +30,12 @@ struct AtmosCache{
     """Timestep of the simulation (in seconds). This is also used by callbacks and tendencies"""
     dt::FT
 
+    """Timestep of the simulation (in seconds). This is also used by callbacks and tendencies"""
```

Comment:
Is this used in tendencies?

Reply:
Whoops, forgot to edit that, I’ll update